app/views/vignettes/slides/edit.html.erb

```diff
@@ -0,0 +1,4 @@
+<!-- app/views/vignettes/slides/edit.html.erb -->
+<h1>Edit Slide</h1>
+
+<%= render 'form' %>
```

Comment:
Create a new slide. Then edit it and click on `Update Slide`. Now, without reloading any page, click on the slide again to edit it. On this edit page, selecting different questions types doesn't work anymore (only if you reload the page manually).